src/vue/common/assets/AssetsList.vue

```diff
@@ -2,11 +2,22 @@
   <div class="assets-list">
     <drawer :is-shown.sync="isDetailsDrawerShown">
       <template slot="heading">
-        {{ 'asset-details.title' | globalize }}
+        <template v-if="isUpdateMode">
+          {{ 'asset-form.update-token-title' | globalize }}
```

Comment:
Is it correct to use translates from another component? `AssetsList` => `'asset-form.update-token-title'`. In the 9th line used `asset-details.title` translate id. In the 38th – `'assets-page'`

Reply:
In that case, titles relate to child components, but are displayed on the parent ones. Now they use translates from child components. 

If the translates must be strongly attached to the their components, I can move them.
@max-nes @letehaha @quinn71

Reply:
@max-nes

Reply:
@letehaha is right, please do not use the translations from one component within another one

Reply:
If the translation comes from a namespace, that is attached to the component, only a component and it's **ONLY** children (I mean if child is used only in that component) **SHOULD** use that translation

P.S. by `attached` I mean having the name of the namespace similar to the name of component, so namespace `asset-form` is attached to `AssetForm`

Reply:
I think it's too complicated to understanding, is it? Also, right now this child component used once in the parent, but in the future, we can reuse this component in another place, but who will remember about this "special case"?

Reply:
Fixed, moved all the assets list translations to `assets-list` namespace